### DISCUSSION TITLE
Request for Sample Input and Output for Verification

### DISCUSSION THREAD

**Discussion by Dikshaingle01** [dayoonkwon/BioAge#42] [Q&A]
Hi Team,

Thank you for sharing this amazing resource and for your work on the BioAge library. I am currently trying to execute the code but want to ensure that my setup and execution are correct.

Could you please provide a sample input of parameters along with the corresponding output? This would help me verify if my execution matches the expected results.

Thank you for your time and support!

Best regards,

**Comment by dayoonkwon** [comment]
The README and `vignettes/examples.Rmd` contain worked examples with sample code and expected output. The NHANES3 and NHANES4 datasets are included in the package, so you can run the examples directly to verify your setup.
